task_opts.go

```diff
@@ -47,6 +47,16 @@ func getCheckpointOptions(r *CheckpointTaskInfo) *runctypes.CheckpointOptions {
 	return opts
 }
 
+// WithEmptyNamespace specifies a namespace which is restored externally
```

Comment:
The word "externally" here means "external to criu". I would change it to say
> ... a namespace which properties are not to be restored by criu

Also maybe add that this corresponds to criu `--empty-ns` option.

Reply:
It doesn't matter what tools are used for C/R